torch/csrc/jit/codegen/fuser/interface.cpp

```diff
@@ -43,6 +43,12 @@ bool canFuseOnGPU() {
 
 void overrideCanFuseOnCPU(bool value) {
   detail::cpu_fuser_enabled = value;
+#ifndef TORCH_ENABLE_LLVM
+  if (value) {
+    throw std::runtime_error(
```

Comment:
Does this kill the process?  I know TORCH_CHECK will propagate cleanly as a RuntimeError to python, so you might switch to that if `runtime_error` kills the python process.

Reply:
I don't see any difference even if I change to `TORCH_CHECK`. Here is the output I see:

```
Process Process-3:
Traceback (most recent call last):
  File "... python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "... python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "perf_regr.py", line 8, in measure
    torch._C._jit_override_can_fuse_on_cpu(fuse)
RuntimeError: Need LLVM support in PyTorch to enable CPU fusion!
```

Reply:
Changed it to `TORCH_CHECK` anyways. It looks more elegant.

Reply:
Cool, thanks for checking!